common/src/test/java/tachyon/master/block/BlockIdTests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.block;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public final class BlockIdTests {
+
+  @Test
+  public void testCreateBlockIdWithMaxSequenceNumber() {
```

Comment:
The convention has been putting the "Test" at the end of the test name. So, this would be "createBlockIdWithMaxSequenceNumberTest()".